src/plugins/mailer/test/index.test.ts

```diff
@@ -1,48 +1,146 @@
-import { FastifyInstance } from 'fastify';
+import { faker } from '@faker-js/faker';
 
-import { SUCCESS_MESSAGES } from '@graasp/translations';
+import { FastifyInstance } from 'fastify';
 
 import build, { clearDatabase } from '../../../../test/app';
 import { resolveDependency } from '../../../di/utils';
+import { MailBuilder } from '../builder';
+import { MAIL } from '../langs/constants';
+import enTranslations from '../langs/en.json';
 import { MailerService } from '../service';
 
 describe('Mailer', () => {
   let app: FastifyInstance;
   let mailerService: MailerService;
+  let mockSendEmail;
 
   beforeAll(async () => {
     ({ app } = await build());
     mailerService = resolveDependency(MailerService);
+    mockSendEmail = jest
+      .spyOn(mailerService, 'sendRaw')
+      .mockImplementation(async () => Promise.resolve());
   });
 
-  afterAll(async () => {
+  afterEach(async () => {
```

Comment:
Split this into an `afterEach` that resets jet mocks and an `afterAll` that resets the DB and closes the server.

Reply:
Done.